### ISSUE TITLE
Project Structure button is broken

### ISSUE THREAD

**Issue by barisyild** [HaxeFoundation/intellij-haxe#1110]
When the Project Structure button is pressed, the panel does not open and the error I mentioned below occurs.

```ex
java.lang.NullPointerException: Cannot invoke "com.intellij.openapi.ui.TextFieldWithBrowseButton.getButton()" because "this.myNekoTextField" is null
	at com.intellij.plugins.haxe.config.sdk.ui.HaxeAdditionalConfigurablePanel.<init>(HaxeAdditionalConfigurablePanel.java:48)
	at com.intellij.plugins.haxe.config.sdk.HaxeAdditionalConfigurable.<init>(HaxeAdditionalConfigurable.java:39)
	at com.intellij.plugins.haxe.config.sdk.HaxeSdkType.createAdditionalDataConfigurable(HaxeSdkType.java:81)
	at com.intellij.openapi.projectRoots.ui.SdkEditor.initAdditionalDataConfigurable(SdkEditor.java:418)
	at com.intellij.openapi.projectRoots.ui.SdkEditor.getAdditionalDataConfigurable(SdkEditor.java:406)
	at com.intellij.openapi.projectRoots.ui.SdkEditor.<init>(SdkEditor.java:100)
	at com.intellij.openapi.roots.ui.configuration.projectRoot.JdkConfigurable.createSdkEditor(JdkConfigurable.java:48)
	at com.intellij.openapi.roots.ui.configuration.projectRoot.JdkConfigurable.<init>(JdkConfigurable.java:38)
	at com.intellij.openapi.roots.ui.configuration.projectRoot.JdkListConfigurable.loadTree(JdkListConfigurable.java:96)
	at com.intellij.openapi.roots.ui.configuration.projectRoot.BaseStructureConfigurable.loadTreeNodes(BaseStructureConfigurable.java:230)
	at com.intellij.openapi.roots.ui.configuration.projectRoot.BaseStructureConfigurable.reset(BaseStructureConfigurable.java:220)
	at com.intellij.openapi.roots.ui.configuration.projectRoot.JdkListConfigurable.reset(JdkListConfigurable.java:138)
	at com.intellij.openapi.roots.ui.configuration.ProjectStructureConfigurable.lambda$reset$1(ProjectStructureConfigurable.java:349)
	at com.intellij.util.io.storage.HeavyProcessLatch.performOperation(HeavyProcessLatch.java:76)
	at com.intellij.openapi.roots.ui.configuration.ProjectStructureConfigurable.reset(ProjectStructureConfigurable.java:335)
	at com.intellij.openapi.options.ex.SingleConfigurableEditor.init(SingleConfigurableEditor.java:59)
	at com.intellij.openapi.options.ex.SingleConfigurableEditor.<init>(SingleConfigurableEditor.java:53)
	at com.intellij.openapi.options.ex.SingleConfigurableEditor.<init>(SingleConfigurableEditor.java:76)
	at com.intellij.openapi.options.ex.SingleConfigurableEditor.<init>(SingleConfigurableEditor.java:88)
	at com.intellij.ide.actions.ShowStructureSettingsAction$1.<init>(ShowStructureSettingsAction.java:41)
	at com.intellij.ide.actions.ShowStructureSettingsAction.showDialog(ShowStructureSettingsAction.java:41)
	at com.intellij.ide.actions.ShowStructureSettingsAction.actionPerformed(ShowStructureSettingsAction.java:37)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.java:327)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$performActionDumbAwareWithCallbacks$4(ActionUtil.java:306)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:350)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks(ActionUtil.java:306)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.lambda$performAction$5(ActionMenuItem.java:296)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.runOnOwnContext(FocusManagerImpl.java:226)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.performAction(ActionMenuItem.java:289)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.lambda$new$0(ActionMenuItem.java:64)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.lambda$fireActionPerformed$4(ActionMenuItem.java:111)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:94)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed(ActionMenuItem.java:111)
	at com.intellij.ui.plaf.beg.BegMenuItemUI.doClick(BegMenuItemUI.java:526)
	at com.intellij.ui.plaf.beg.BegMenuItemUI$MyMouseInputHandler.mouseReleased(BegMenuItemUI.java:558)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6648)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3392)
	at java.desktop/java.awt.Component.processEvent(Component.java:6413)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5022)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4948)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4575)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4516)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2802)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4854)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:730)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:724)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:754)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:752)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:751)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:909)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.java:831)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:753)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$5(IdeEventQueue.java:437)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:436)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:615)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:434)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:838)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:480)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```

**Comment by m0rkeulv** [comment]
Hopefully this is fixed in the latest 2023.1 Beta, let me know if it does not solve the problem
https://github.com/HaxeFoundation/intellij-haxe/releases/tag/pre-release%2F202304231701

**Comment by barisyild** [comment]
it looks like the problem is solved!
